tests/cases/fourslash/quickInfoForObjectBindingElementName04.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference path="fourslash.ts" />
+
+////interface Options {
+////   /**
+////    * A description of 'a'
+////    */
+////    a: {
+////       /**
+////        * A description of 'b'
+////        */
+////       b: string;
+////   }
+////}
+////
+////function f({ a, a: { b } }: Options) {
```

Comment:
can you add a test of the union case showing that it only takes the first declaration's comment?